crates/ruff_python_formatter/src/expression/string/docstring.rs

```diff
@@ -1,6 +1,7 @@
 // This gives tons of false positives in this file because of
 // "reStructuredText."
 #![allow(clippy::doc_markdown)]
+#![allow(warnings)]
```

Comment:
all of them? Is this the start of a rebellion against our pedantic rules (I consider joining :P)

Reply:
Oh my heavens no, haha. No rebellion here. I sometimes add this to the top of a file when adding a new chunk of code. I just forgot to remove it.

I do find some of our Clippy lints a touch pedantic, but no strong feelings. Yet.